bears/general/IndentationBear.py

```diff
@@ -12,6 +12,12 @@
 
 class IndentationBear(LocalBear):
 
+    LANGUAGES = {"All"}
```

Comment:
but we used ALL for all general bears, so if we do that change, we should not have this attribute then and do the generator thingie in a whole new commit changing all of them :)